v1/api/swagger/components/schemas/DiskPartition.yaml

```
@@ -0,0 +1,30 @@
+type: object
+description: Parition to be created on the host.
```

Typo 'parition'. This is a common misspelling. Might want to grep for other instances.
```suggestion
description: Disk partition to be created on the host.
```